### ISSUE TITLE
Pre-fill app setup modal

### ISSUE THREAD

**Issue by joebudi** [Budibase/budibase#5188]
**Describe the feature request**
I feel we should pre-fill the app creation modal with a sensible default (that increments)

For example:

Name:
`My first app`

Url:
`my-first-app`

Twilio and others do something like this, and it makes onboarding faster and removes potential obstacles. 

**Screenshots**
<img width="1676" alt="CleanShot 2022-03-31 at 11 04 42@2x" src="https://user-images.githubusercontent.com/49767913/161030674-638cda1c-8147-4f10-aedf-b84c35b12a1e.png">

<img width="1679" alt="CleanShot 2022-03-31 at 11 05 05@2x" src="https://user-images.githubusercontent.com/49767913/161030755-32ca821e-2760-45c6-bd85-004346586066.png">

<img width="1920" alt="CleanShot 2022-03-31 at 10 57 40@2x" src="https://user-images.githubusercontent.com/49767913/161030475-6deb172b-a1ea-4020-8154-9a67e273f47a.png">

**Notes/further work**
I would add to this, with the new designs, we will display the app name within the app-building UI. The user should be able to change the app/domain of the app here in my opinion (@AndyWhann ) what do you think?

**Comment by AndyWhann** [comment]
Yeah we could do this but the field needs to have the text highlighted and in focus by default so the user can delete this suggestion easily too.

Would also maybe suggest using the name of the user e.g 

Andy's app (1)

**Comment by aptkingston** [comment]
This is already done for templates at least, Dean added it. Name is focused automatically too.

e.g. when clicking on a template:
![image](https://user-images.githubusercontent.com/9075550/161039666-48072dbf-90f4-41d4-bfe3-53b6455c690f.png)
